src/test/java/mate/academy/service/RoleServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.service;
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import mate.academy.dao.RoleDao;
+import mate.academy.model.Role;
+import mate.academy.service.impl.RoleServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class RoleServiceTest {
```

Comment:
RoleService is a pretty simple class. There is no need to write tests for it because in this test class you are calling methods in mock objects which makes no sense. Same for UserService

Reply:
Ok, I get it)